tailwind.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  content: [
+    
+      "./index.html",
+      "./src/**/*.{vue,js,ts,jsx,tsx}",
+    
+  ],
+  theme: {
+    extend: {},
+  },
+  plugins: [require('@tailwindcss/forms'),],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/bfoujols/relearun/pullRequest?prid=8781771)
```suggestion
  plugins: [require("@tailwindcss/forms"),],
```